app/src/test/java/com/mobiledevelopmentfoodapp/com/mobiledevelopmentfoodapp/FoodTest.kt

```diff
@@ -5,7 +5,7 @@ import org.junit.Test
 
 class FoodTest {
     @Test
-    fun addition_isCorrect() {
+    fun `addition is correct`() {
```

Comment:
renamed test function to be used within back ticks to adhere to https://kotlinlang.org/docs/coding-conventions.html#function-names